frontend/src/pages/workspace/WorkspacePage.tsx

```diff
@@ -1,13 +1,21 @@
+import { useQuery } from 'react-query'
+import { Divider } from '@chakra-ui/react'
+
 import { useAuth } from '~contexts/AuthContext'
+import { getLandingPageStatistics } from '~services/AnalyticsService'
 import Button from '~components/Button'
 
 export const WorkspacePage = (): JSX.Element => {
   const { user, logout } = useAuth()
 
+  const { data, isLoading } = useQuery('stats', getLandingPageStatistics)
```

Comment:
assuming this is temporary and we'll wrap this query in a custom hook when we actually implement it

Reply:
correct, this is just to test whether the frontend is correctly calling backend APIs